eval/bundled/epm.elv.go

```diff
@@ -111,7 +111,12 @@ fn tilde-expand [p]{
     &upgrade= [pkg dom-cfg]{
       dest = (dest $pkg)
       -info "Updating "$pkg
-      git -C $dest pull
```

Comment:
what about using `try`:

```
try {
  git -C $dest pull
} except _ {
  -error "git pull failed"
}
```

Reply:
Of course! Much simpler code. Thanks for the feedback - done.